src/graph.rs

```diff
@@ -0,0 +1,98 @@
+//! This module facilitates the building of new named graphs as well as the retrieval
+//! of existing indexes in ArangoDB.
+//!
+//! The various structures are following the HTTP specification as detailed in this ArangoDB
+//! [section](https://www.arangodb.com/docs/stable/http/gharial-management.html)
+//!
+//! For detailed information about ArangoDB named graphs, please check out the official
+//! ArangoDB [documentation](https://www.arangodb.com/docs/stable/http/gharial.html).
+use serde::{Deserialize, Serialize};
+use typed_builder::TypedBuilder;
+
+pub const GHARIAL_API_PATH: &str = "_api/gharial";
```

Comment:
Is `GHARIAL_API_PATH` a typo? Or am I missed something?

Reply:
For some reason it's called [gharial](https://www.arangodb.com/docs/stable/http/gharial-management.html) but I can rename it to "graph" for the `const`.

I changed both new `const` into `pub(crate)` as you requested

Reply:
That is fine, don't need to rename. Thank you.